### ISSUE TITLE
Use mc-router with bedrock / udp ?

### ISSUE THREAD

**Issue by euri10** [itzg/mc-router#14]
Greetings,

First thanks for the amazing image(s), it's been a seamless experience and so easy to use.

I'm using for my son and his friends a vanilla server running `itzg/minecraft-server`.

Now some of his friends have bedrock only so I managed to transfer the whole world / players on a Paper server using still your amazing image, and I installed the [Geyser](https://ci.nukkitx.com/job/GeyserMC/job/Geyser/job/master/) plugin which works fine.

I can expose the 19132 udp port in the container (that's what geyser uses) and bedrock clients connect to my ip:19132 fine and "regular" users can connect to `regular.mydomain` with the mc-router `--mapping=regular.mydomain=my_paper_container:25565`

Now I'd like to 
1. not expose 19132 port in `my_paper_container`
2. map this port thanks to mc-router

I tried to add another subdomain `bedrock.mydomain=my_paper_container:19132` without any luck, is it possible ? I tried a few variations like `bedrock.mydomain=my_paper_container:19132/udp`

in summary this compose file works
```
version: "3.7"
services:
  my_paper_container:
    image: itzg/minecraft-server
    restart: always
    environment:
      - TYPE=PAPER
      - EULA=TRUE
      - VERSION=1.16.4
      - MEMORY=3G
    ports:
      - 19132:19132/udp
  router:
    image: itzg/mc-router
    restart: always
    ports:
      - 25565:25565
    command: --mapping=regular.mydomain=my_paper_container:25565
    #command: --mapping=bedorck.mydomain=my_paper_container:19132/udp,regular.mydomain=my_paper_container:25565
```

what I'd like is to comment the `my_paper_container` ports section and uncomment the `--mapping` in mc-router !

hope this is clear, let me know if that sounds sensible ?

**Comment by euri10** [comment]
oups sorry it's working as intended, not sure why I couldnt connect before

**Comment by itzg** [comment]
I'm glad you got it working...but I'm not quite sure how it's working since mc-router only knows how to speak the Java edition protocol during handshaking.
